examples/indexer.rs

```diff
@@ -35,11 +40,13 @@ async fn main() -> Result<()> {
 
     println!(
         "{}",
-        request_funds_from_faucet("http://localhost:8091/api/enqueue", &address.to_bech32("atoi"),).await?
+        request_funds_from_faucet(&faucet_url, &address.to_bech32(SHIMMER_TESTNET_BECH32_HRP)).await?
```

Comment:
How about not even using the `SHIMMER_TESTNET_BECH32_HRP` constant, but getting it from the nodeinfo? 
`client.get_bech32_hrp().await?` can be used for that

Reply:
sure